subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_dependencies.adoc

```diff
@@ -209,6 +209,13 @@ link:{javadocPath}/org/gradle/api/file/ProjectLayout.html#files-java.lang.Object
 and link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:fileTree(java.lang.Object)[Project.fileTree(java.lang.Object)]
 Alternatively, you can also define the source directory of one or many file dependencies in the form of a <<declaring_repositories.adoc#sub:flat_dir_resolver,flat directory repository>>.
 
+[NOTE]
+====
+A `FileTree` ordering is OS and filesystem sensitive.
```

Comment:
I wonder if it is `A FileTree ordering`, `The FileTree ordering` or just `FileTree ordering`.

Maybe this is better:
`The order of files in a fileTree is OS and filesystem dependent`.

Note that even that is not 100% true, since the order depends on the machine as well, e.g. on two Linux machines with the same filesystem, the order may be different.

Reply:
Ok, I will update the text to effectively state that it has no stable ordering at all. Thanks for the review!